### PR TITLE
NOISSUE - Enable Standalone Mode For Things

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
It enables standalone mode for things i.e things can run independently from users if configured so.

### Which issue(s) does this PR fix/relate to?
Noissue

### List any changes that modify/break current functionality
Enables functionality of running things in standalone mode without `auth` service

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please resolve conflicts.

File: docker/.env [hunk 1]
```diff
@@ -87,6 +87,8 @@ MF_THINGS_ES_URL=localhost:6379
 MF_THINGS_ES_PASS=
 MF_THINGS_ES_DB=0
 MF_THINGS_URL=http://mainflux-things:9000
+MF_THINGS_STANDALONE_ID=
+MF_THINGS_STANDALONE_TOKEN=
```
Comment (SammyOina) [inline docker/.env:91]: Do these have defaults

Comment (rodneyosodo) [reply docker/.env:91]: No. When configuring is when you set them.

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Update README.md, as well.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**